src/main.rs

```diff
@@ -48,10 +53,21 @@ enum Commands {
 fn main() {
     let cli = Cli::parse();
 
-    let ssh_user = cli.user.unwrap_or(env::var("USER").unwrap());
+    let ssh_user = match cli.user {
+        Some(user) => user,
+        None => (|| -> Option<String> {
```

Comment:
this looks a bit too complicated. I don't think an additional function is needed here as it could just be done in the match case itself?

Reply:
I did it this way, so I could just do unwrap on it and not worry about error handling. Do you have a better suggestion?

Reply:
see #4